cypress/support/cypress-commands/commands.js

```diff
@@ -1549,3 +1548,23 @@ Cypress.Commands.add('startOrStopInteractionsService', (action) => {
 Cypress.Commands.add('envConfigSetup', () => {
   fireLog.info('No additional config module environment setup');
 });
+
```

Comment:
comments and function descriptions are missing

Reply:
added